src/templates/withDefaultContext.ts

```diff
@@ -1,6 +1,8 @@
 import * as intl from 'intl';
 import { defaults } from 'lodash';
-import * as moment from 'moment';
+
+// tslint:disable-next-line:no-var-requires
+const { formatToTimeZone } = require('date-fns-timezone');
```

Comment:
Why not import?

Reply:
Agree, types are included in `date-fns-timezone`. No need for disable nor require.

Reply:
:thinking: i dont know why i used this

fixed